articles/microsites/call-api/call-api-single-page-app.md

```diff
@@ -21,37 +21,35 @@ Your user will authenticate, and Auth0 will generate an ID Token and Access Toke
 
 When your app needs to fetch user data from your API:
 
-1. If the user is not already authenticated, our SDK redirects the user to your Auth0 Authorization Server, requesting an ID Token.
-2. The user authenticates with Auth0 using one of your configured login options (e.g., username/password, social identity provider, <dfn data-key="security-assertion-markup-language">SAML</dfn>), 
-3. Auth0 responds with the user's ID Token and Access Token.
-3. The Access Token can be used to call your API and retrieve requested data.
+1. If the user is not already authenticated, our SDK redirects the user to your Auth0 Authorization Server.
+2. The user authenticates with Auth0 using one of your configured login options (e.g., username/password, social identity provider, <dfn data-key="security-assertion-markup-language">SAML</dfn>).
+3. Your app requests an ID Token and Access Token.
+4. Auth0 responds with the requested tokens.
+5. The Access Token can be used to call your API and retrieve requested data.
 
-For single-page web apps, Auth0 uses the [Implicit Flow](/flows/concepts/implicit).
+For single-page apps, Auth0 uses the [Authorization Code Flow with PKCE](/flows/concepts/auth-code-pkce).
 
-<img src="/media/articles/microsites/overview-flow-call-api-single-page-app.png" alt="Flow Overview for Single-Page Apps" width="100%">
+<img src="/media/articles/microsites/overview-flow-call-api-single-page-app-pkce.png" alt="Flow Overview for Single-Page Apps with Auth Code Flow with PKCE" width="100%">
```

Comment:
This image only shows /authorize endpoint which would assume implicit.  We should swap for a code grant workflow